libnmstate/validator.py

```diff
@@ -33,14 +34,34 @@
 from libnmstate.schema import InterfaceState
 from libnmstate.schema import LinuxBridge as LB
 from libnmstate.schema import OVSBridge
+from libnmstate.schema import Team
 from libnmstate.schema import VXLAN
-from libnmstate.schema import Bond
 from libnmstate.error import NmstateDependencyError
 from libnmstate.error import NmstateNotImplementedError
 from libnmstate.error import NmstateValueError
 from libnmstate.iplib import is_ipv6_address
 
 
+INTERFACE_SLAVE_TREES = {
```

Comment:
In the `libnmstate/appliers/` foler, there are `get_slaves_from_state()` `get_bond_slaves_from_state()` and etc.

Please use these functions instead.

Reply:
I didn't know those existed. Thanks!
Now it is implemented with that. The IPV6 tests were also added.